Nodejs/Product/Nodejs/Repl/NodejsReplEvaluator.cs

```diff
@@ -39,6 +39,8 @@ public NodejsReplEvaluator(INodejsReplSite site)
             this._site = site;
         }
 
+        public string NodeExePath { get; private set; }
```

Comment:
Shouldn't this compute and then cache `GetNodeExePath`?  Why would that be the caller's responsibility?